Apps/WebClient/src/appsettings.json

```diff
@@ -10,7 +10,7 @@
         "ClientId": "healthgateway",
         "Audience": "healthgateway",
         "ResponseType": "code",
-        "Scope": "openid profile web-origins",
+        "Scope": "openid profile web-origins email",
```

Comment:
not necessary for this PR, but will be in dev anyway

Reply:
This may conflict with the same change that was made in https://github.com/bcgov/healthgateway/pull/2629/commits/5d26efbb597c82214b190aa9b275165afb6c490d.

Reply:
should be the same change so I think we are ok